### PR TITLE
koord-scheduler: ignore non-existing resources in NUMA Node

### PR DESCRIPTION
**Pull request by eahydra**
### Ⅰ. Describe what this PR does

At present, koordlet only reports cpu/memory in NodeResourceTopology. If a Pod requests resources from other resources, the Pod will not be able to be scheduled. So we should skip these non-existing resources.

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: pkg/scheduler/plugins/nodenumaresource/resource_manager.go [hunk 1]
```diff
@@ -207,10 +212,17 @@ func (c *resourceManager) allocateResourcesByHint(node *corev1.Node, pod *corev1
 			break
 		}
 	}
-	requests = quotav1.RemoveZeros(requests)
-	if len(requests) > 0 {
-		resourceNames := quotav1.ResourceNames(requests)
-		return nil, fmt.Errorf("insufficient resources: %v", resourceNames)
+
+	var insufficientResources []string
```
Comment (hormes) [inline pkg/scheduler/plugins/nodenumaresource/resource_manager.go:216]: Add a unittests to verify this？

Comment (eahydra) [reply pkg/scheduler/plugins/nodenumaresource/resource_manager.go:216]: done

**Review by eahydra — COMMENTED** [review]
/approve

**Pull request merged**